corehq/apps/ota/tests/test_case_restore.py

```diff
@@ -103,8 +102,43 @@ def test_locations_in_restore(self):
         self.assertXmlPartialEqual(locations_content, response_content,
                                    '{http://openrosa.org/http/response}fixture[@id="locations"]')
 
-    def _generate_restore(self, case_id, user):
-        self.client.login(username=user.username, password=user.password)
+    @flag_enabled('ADD_LIMITED_FIXTURES_TO_CASE_RESTORE')
+    def test_lookup_table_in_restore(self):
+        case_id = self.dad.case_id
+
+        domain_obj = create_domain(self.domain)
+
+        self.addCleanup(delete_all_users)
+        self.addCleanup(domain_obj.delete)
+
+        table_tag = "atable"
+
+        table = LookupTable(domain=self.domain,
+                            tag=table_tag,
+                            description="A Table",
+                            is_global=True,
+                            fields=[TypeField(name="wing")])
+        table.save()
+        row = LookupTableRow(
+            table_id=table.id,
+            domain=self.domain,
+            fields={
+                "wing": [Field(value="duck", properties={"says": "quack"})],
+            },
+            sort_key=0,
+        )
+        row.save()
+
+        response = self._generate_restore(case_id)
+        self.assertEqual(response.status_code, 200)
+
+        response_content = next(response.streaming_content)
+
+        lookup_table_content = lookup_table_fixture_content.format(table_tag=table_tag, user_id=case_id)
+        xpath = '{h}fixture[@id="item-list:{tag}"]'.format(h='{http://openrosa.org/http/response}', tag=table_tag)
```

Comment:
Nit: to use curly braces in a format string without them getting interpreted as a template variable, you can double them up, like so:
```suggestion
        xpath = f'{{http://openrosa.org/http/response}}fixture[@id="item-list:{table_tag}"]'
```